### PR TITLE
Adjust padding-top based on navbar height

### PR DESCRIPTION
**Pull request by dpprdan**
For both the tidytemplate and the rotemplate packages there are issues where the navbar overlaps the page body on small screens (https://github.com/tidyverse/tidytemplate/issues/42, https://github.com/ropensci/rotemplate/issues/15). 

In principle, this problem also exists with the default pkgdown theme, though not as often, because of the smaller navbar font size. Here are two examples [in the wild](https://radiant-rstats.github.io/radiant.multivariate/):

radiant_ipad

radiant_300

This can be fixed if the top-padding of the body is set relative to the navbar, see [this answer on SO](https://stackoverflow.com/a/20197508/2114932), which this PR draws upon.

My knowledge of Javascript and CSS is extremely limited, so it's quite likely that this PR can be improved upon, hence my labelling it as proof-of-concept. 

With this PR applied, pkgdown shows the whole headline of the page body (using a dummy package here, since I do not have all radiant.multivariate dependencies installed).

pkg_long_name

### CODE REVIEW

**Comment by dpprdan** [comment]
FWIW headroom (#1032) does not solve this. 
This is with current master (e8a2d7176cd78a375a1b16fc77113b91ce5413cb)
pkgdown_master

This with this PR applied
pkgdown_padding-top

Of course it would be better to tweak the navbar so that it does not cover more than one line, e.g. hide the version number in this case. So this PR is meant to provide a safe fall-back.

File: inst/templates/navbar.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {{#navbar}}
-<div class="navbar navbar-{{{type}}} navbar-fixed-top" role="navigation">
+<div class="navbar navbar-{{{type}}} navbar-fixed-top" role="navigation" id="topnavbar">
```
Comment (jayhesselberth) [inline inst/templates/navbar.html]: Why do you need to add a new `#topnavbar` here? I think you can just change `#topnavbar` to `.navbar` in the JS code above.

File: inst/assets/pkgdown.js [hunk 2]
```diff
@@ -39,6 +39,13 @@
     }
   });
 
+  $(function() {
```
Comment (jayhesselberth) [inline inst/assets/pkgdown.js]: I don't think this needs to be in its own `function` block. You can move the following to the top after the headroom call. Note that I updated to refer to the `navbar` class. 

```js
$(document.body).css('padding-top', $('.navbar').height() + 10);
$(window).resize(function(){
  $(document.body).css('padding-top', $('.navbar').height() + 10);
});

```

**Comment by dpprdan** [comment]
Thanks @jayhesselberth, I simplified the JS and HTML as per your suggestions (and used `$('body').css` as in `$('body').scrollspy`).

I noticed that the `?build_home()` suggests that the rendered homepage shows the package logo (if present) and calls `init_site` for the favicons (and hence copy `pkgdown.js` et.al. into `docs/dev`). That does not happen, however. Should I open an issue for that?

**Comment by jayhesselberth** [comment]
Thanks!

**Pull request merged**